### ISSUE TITLE
SplitButton does not show correctly in IE and Firefox 

### ISSUE THREAD

**Issue by 007304wz** [primefaces/primeng#5061]
### There is no guarantee in receiving an immediate response in GitHub Issue Tracker, If you'd like to secure our response, you may consider *PrimeNG PRO Support* where support is provided within 4 business hours

**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
https://www.primefaces.org/primeng/#/splitbutton (open in IE)

**Expected behavior**
https://www.primefaces.org/primeng/#/splitbutton (open in Chrome)

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser Firefox, IE

**Comment by jupin-r** [comment]
The Microsoft Edge does not show correctly SplitButton too.

**Comment by Merve7** [comment]
Fixed in 5.2.4
